docs/governance/tags-overview.md

```diff
@@ -13,17 +13,21 @@ ms.date: 05/05/2025
 
 # Tags in Microsoft Fabric
 
-Tags is a Microsoft Fabric feature that gives organizations the ability to apply additional metadata to items in Fabric to facilitate and enhance data categorization, organization, and discoverability. Tags are configurable text labels, such as *Sales – FR 2023*, *HR – Summer Event*, or *FY 2024*, that Fabric administrators can define according to their organization's needs. Data and content owners can then apply these tags to their Fabric items, and users in the organization can use the tags to help them find the data and content they need.
+Tags is a Microsoft Fabric feature that gives organizations the ability to apply additional metadata to items in Fabric to facilitate and enhance data categorization, organization, and discoverability. Tags are configurable text labels, such as *Sales – FR 2025*, *HR – Summer Event*, or *FY 2025*, supported at two organizational levels: tenant and [domain](./domains.md) level, providing flexibility in how assets are governed across your organization. Data and content owners can then apply these tags to their Fabric items, and users in the organization can use the tags to help them find the data and content they need.
```

Comment:
Line 16 suggestion:

Tags is a Microsoft Fabric feature that gives organizations the ability to apply additional metadata to items in Fabric to facilitate and enhance data categorization, organization, and discoverability. Tags are configurable text labels, such as *Sales – FR 2025*, *HR – Summer Event*, or *FY 2025*, that can be applied at both the tenant and [domain](./domains.md) levels, offering flexibility in how assets are governed across your organization. Data and content owners can then apply these tags to their Fabric items, and users in the organization can use the tags to help them find the data and content they need.